src/traffic_server/InkAPI.cc

```diff
@@ -792,6 +792,23 @@ sdk_free_field_handle(TSMBuffer bufp, MIMEFieldSDKHandle *field_handle)
   }
 }
 
+/**
```

Comment:
Really need a `static_assert` here to verify `sizeof(TSHdrHandle) == sizeof(MIMEFieldSDKHandle)`